PythonPonies/bangazon-api
# related hyperlinks established

# Description
Add description about what this pull request covers.
Establishes a hyperlink relationship between the order and the products on order models so that orders are now listed as hyperlinks on the products on order model.

## Number of Fixes
Type of number of fixes included in this PR.
No fixes, but there is new functionality.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#78 #69 

## Problem to Solve
Describe the problem(s) this pull request solves.
We need this step to be able to properly display product information on the order. We will have to continue to revise in order to show the proper fields, but the relationship has been established. 

## Proposed Changes
Describe the proposed changes.
As per the technical manager's suggestion, we'll need to investigate 'depth' to see if we can get the right information to display.

## Expected Behavior
Describe the expected behavior.
A url for each of the associated products should display under the proper order. When clicked, it should take you to product information. 

## Steps to Test Solution
1. Git fetch this branch (ls-tinker)
2. Python manage.py makemigrations and then manage.py migrate
3. Run the server and navigate to the api root url.
4. Click on the orders url and check for associated products under order 3. 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
